src/WPF/ArcGISRuntime.WPF.Samples/Samples/MapView/FeatureLayerTimeOffset/FeatureLayerTimeOffset.xaml.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using System;
+
+namespace ArcGISRuntime.WPF.Samples.FeatureLayerTimeOffset
+{
+    public partial class FeatureLayerTimeOffset
+    {
+        // Hold the feature layer URI
+        private Uri _featureLayerUri = new Uri("https://sampleserver6.arcgisonline.com/arcgis/rest/services/Hurricanes/MapServer/0");
+
+        // Hold a reference to the original time extent
+        private TimeExtent originalExtent;
+
+        public FeatureLayerTimeOffset()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create new Map
+            Map myMap = new Map(Basemap.CreateOceans());
+
+            // Create the hurricanes feature layer once
+            FeatureLayer noOffsetLayer = new FeatureLayer(_featureLayerUri);
+
+            // Apply a blue dot renderer to distinguish hurricanes without offsets
+            noOffsetLayer.Renderer = new SimpleRenderer(new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Circle, System.Windows.Media.Colors.Blue, 10));
+
+            // Add the non-offset layer to the map
+            myMap.OperationalLayers.Add(noOffsetLayer);
+
+            // Create the offset hurricanes feature layer
+            FeatureLayer withOffsetLayer = new FeatureLayer(_featureLayerUri);
+
+            // Apply a red dot renderer to distinguish these hurricanes from the non-offset hurricanes
+            withOffsetLayer.Renderer = new SimpleRenderer(new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Circle, System.Windows.Media.Colors.Red, 10));
+
+            // Apply the time offset (red hurricane dots will be from 10 days before the current extent)
+            withOffsetLayer.TimeOffset = new TimeValue(10, Esri.ArcGISRuntime.ArcGISServices.TimeUnit.Days);
```

Comment:
The code would read cleaner, if this line were broken into 2 lines of code. One line of code for the creation of the TimeValue and another to set the FeatureLayer.TimeOffset property. Chaining of code together makes it harder to understand what is occurring and requires a lot of scrolling to read what is going on.